test/manipulation.test.js

```diff
@@ -659,10 +659,60 @@ describe('manipulation', function() {
   });
 
   describe('updateOrCreate', function() {
+    var db = getSchema();
+    var Todo;
+
+    before(function setUp(done) {
+      Todo = db.define('Todo', {
+        content: String,
+      });
+      db.automigrate('Todo', done);
```

Comment:
Note that `automigrate` may be rather expensive and by calling it before each test, you can slow the test suite. I think a better solution is to 1) call automigrate on the first run (before the first test case) and then 2) delete all model instances on subsequent runs.

Alternatively, you can split this hook into two parts:
 - `before` hook to setup the model and automigrate the database
 - `beforeEach` hook to delete all model instances